### ISSUE TITLE
font-lock-doc-face unspecified

### ISSUE THREAD

**Issue by jordonbiondo** [cmack/emacs-planet-theme#3]
I've been using a few different colors for font-lock-doc-face trying to find the best one, one solution, and maybe the best solution is to just use the same color as font-lock-comment-face, but in the gif are the three I've been considering using. 
![planet-doc-colors](https://cloud.githubusercontent.com/assets/1441728/2591707/3787b028-ba6e-11e3-90c7-786fe573e7b5.gif)

**Comment by cmack** [comment]
Sorry, I've decided that I find many of the default inheritance rules a better fit. With ed544891d4 I've added the missing font-lock faces explicitly to make sure they aren't over looked for potential future changes. I did decide to add the `font-lock-negation-char-face` using `planet-warning` hue.

Visually diminishing `font-lock-comment-delimiter` is a non-starter for me as some languages have extra context supplied by the comment delimiters (e.g. Lisp's `;;;;` vs `;;`)
